test/T44.stl.names.hpp

```diff
@@ -22,20 +23,15 @@
 #include <unordered_set>
 #include <unordered_map>
 
-void foo(std::set<float> ) {}
 void foo(std::list<double> ) {}
 void foo(std::forward_list<float> ) {}
-void foo(std::map<float,int> ) {}
 void foo(std::vector<double> ) {}
 void foo(std::deque<int> ) {}
```

Comment:
Why not move these test cases as well? That way we can have 42 only for `multi*`. 

If you agree with above then lets also rename 42 to `T42.stl.names.multi.hpp` to make reason for this separation clear. (and it might be a good idea to put some minimal comment into both files explaining and referencing this PR for clarity). 

Thanks,